pkg/volume/csi/proto/csi/csi.pb.go

```diff
@@ -0,0 +1,2548 @@
+/*
```

Comment:
It is better to remove it and refer from the vendor directory.

Reply:
@edisonxiang yes, will do.